SteamReview/SteamReview/CategoryContainer.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <iostream>
+#include "Category.h"
+
+class FCategoryContainer
+{
+private:
+	static const int MaxCategoryCount = 6;
+	FCategory Categories[MaxCategoryCount];
+	int CategoryCount;
+public:
+	FCategoryContainer();
+	bool AddCategory(const FCategory InCategory);
+	bool DeleteCategory(const int InCategoryIndex);
+	FCategory GetCategory(const int InCategoryIndex) const;
+	bool AddGameToCategory(const int IndexCategory, const FGame InGame);
+	void PrintCategory() const;
+	int GetCurrentCategory() const;
```

Comment:
Nice! It's very useful to differentiate between the actual object and a count variable. I agree that `GetCurrentCategory()` is not an appropriate name, since it sounds as if it would return a category rather than an int.